pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -3346,6 +3346,17 @@ final: prev:
     meta.homepage = "https://github.com/junegunn/gv.vim/";
   };
 
+  hare-vim = buildVimPluginFrom2Nix {
+    pname = "hare.vim";
+    version = "2022-07-02";
+    src = fetchgit {
+            url = "https://git.sr.ht/~sircmpwn/hare.vim";
+            rev = "5c758cdbbabd6e4ba92bced9428cd1fa4212f003";
+            sha256 = "0l1q9x1n30h32zfnd2krg3frywydfmb8ic05619pylamnyh4w918";
+        };
```

Comment:
Please use [`fetchFromSourceHut`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchsourcehut/default.nix) and make sure you indent it out with 2 spaces.

Reply:
Fixed. Not sure how I messed up indentation; I'd used the `update.py` script to generate this.

Reply:
It's probably the script that doesn't indent properly. To use [fetchFromSourcehut](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchsourcehut/default.nix), one would need to change the updater  script too.

Reply:
Honestly, I prefer seeing the URL here. It's easy to copy-paste to find upstream. Using `fetchFromSourceHut` obfuscates the URL a lot.